bluetooth-audio/Dockerfile.aarch64

```
@@ -1,17 +1,9 @@
-FROM balenalib/raspberrypi3:stretch
+FROM balenalib/raspberrypi3-debian:stretch
 
 ENV DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket
-
 ENV UDEV=1
 
-RUN install_packages \
-  alsa-utils \
-  bluealsa \
-  bluez \
-  python-gobject \
-  python-dbus \
-  python-gpiozero \
-  mplayer
+RUN install_packages alsa-utils bluealsa bluez python-gobject python-dbus python-gpiozero mplayer
```

```suggestion
RUN install_packages alsa-utils bluealsa bluez python-gobject python-dbus python-gpiozero mplayer
```

What's the reason for changing this? One per line is more readable, no? They should be 1 per line and also alphanumerically sorted (which granted they weren't before), according to [Dockerfile best practises](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/).

---

TIL about this best practice. I was just going by my preferred way of formatting it. Will fix it!